src/page-objects/AdministrationPages.ts

```diff
@@ -84,10 +96,26 @@ export const test = base.extend<FixtureTypes>({
         await use(new OrderDetail(AdminPage));
     },
 
+    AdminCustomerListing: async ({ AdminPage }, use) => {
+        await use(new CustomerListing(AdminPage));
+    },
+
     AdminCustomerDetail: async ({ AdminPage }, use) => {
         await use(new CustomerDetail(AdminPage));
     },
 
+    AdminCustomerGroup: async ({ AdminPage }, use) => {
```

Comment:
we talk here about the CustomerGroupListing or? Please change it properly

Reply:
Changed it.